docs/api/modules/App-Configuration.md

```diff
@@ -204,13 +208,42 @@ if (!global.BROWSER) {
       escapeHatch: false,
       // we can optionally define a scope to use with the service worker
       scope: '/',
+      // the web app manifest can be directly incorporated in the PWA config
+      webManifest: {
+        // the full name is the official name of a given PWA
+        name: 'My App Name',
+        // the short name is used by mobile devices to label your home screen icon
+        short_name: 'My App',
+        // the description is a good piece of meta-data to include for a short description
+        // which can be used with presenting your PWA
+        description: 'My PWA app.',
```

Comment:
Can you use all the features listed here https://developer.mozilla.org/en-US/docs/Web/Manifest or on those listed here?
 From the article i noticed that `icons` is mandatory is mandatory. Do we have some features that are required for this setup?

Reply:
Yes, we can use the features listed on both sources! I based the validation to account for all the keys present on https://developer.mozilla.org/en-US/docs/Web/Manifest , although key difference is we only require the `"name"` field and not necessarily `"icons"` at the moment. We can wrap the `icon` validation in `createIsRequired` if we desire.

Reply:
Awesome, thank you! This looks amazing. Great work Tobia

Reply:
Maybe we can mention that `name` is required